docker-compose-prod.yml

```diff
@@ -20,6 +20,7 @@ services:
     volumes:
       - ./webapplication:/code
       - ../data:/data
+      - ../data/media:/code/media
```

Comment:
Better not mix ./webapplication -> /code and ../data/media -> /code/media , as it depends on the order of mounting.
Mount ../data/media to /media

Reply:
fixed